src/assets/contributors-map.geojson

```diff
@@ -150,6 +150,21 @@
           35.198478054480816
         ]
       }
-  }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "firstName": "Melissa",
+        "githubId": "melissapthai",
+        "favoriteCrag": "8aa15dd0-878e-5a34-ad5d-bd1f96f2e17e"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          39.98884394309755,
+          -105.28787643505346
```

Comment:
You'll have to swap the order of the coordinates.  This is a very common recurrence.

Reply:
Done!